README.md

```diff
@@ -81,17 +96,35 @@ Configure settings in the /etc/cockpit/cockpit.conf file.  See [`man cockpit.con
     cockpit_port: 9090
 Cockpit runs on port 9090 by default. You can change the port with this option.
 
-Note that the default SELinux policy does not allow Cockpit to listen to anything else than port 9090, so you need to allow that first, with e.g.
+    cockpit_manage_firewall: false
+Boolean flag allowing to configure firewall using the firewall role.
+Manage the cockpit firewall service.
+If the variable is set to `false`, the `cockpit role` does not manage the firewall.
+Default to `false`.
+
+NOTE: `cockpit_manage_firewall` is limited to *adding* ports.
+It cannot be used for *removing* ports.
+If you want to remove ports, you will need to use the firewall system
+role directly.
 
-    semanage port -m -t websm_port_t -p tcp 443
+NOTE: This functionality is supported only when the managed host's `os_family`
+is `RedHat`.
 
-for ports that are already defined in the SELinux policy, such as 443, or
+    cockpit_manage_selinux: false
+Boolean flag allowing to configure selinux using the selinux role.
+Assign `websm_port_t` to the port.
+If the variable is set to false, the `cockpit role` does not manage the
+selinux
```

Comment:
"the selinux" is ungrammatical. "SELinux" or "the `selinux` role", or perhaps "SELinux permissions of the cockpit port".